pkg/standalone/standalone.go

```diff
@@ -693,19 +699,19 @@ func createSlimConfiguration(wg *sync.WaitGroup, errorChan chan<- error, info in
 	}
 }
 
-func makeDefaultComponentsDir() error {
+func makeDefaultResourcesDir() error {
 	// Make default components directory.
-	componentsDir := DefaultComponentsDirPath()
+	resourcesDir := DefaultResourcesDirPath()
 	//nolint
-	_, err := os.Stat(componentsDir)
+	_, err := os.Stat(resourcesDir)
 	if os.IsNotExist(err) {
-		errDir := os.MkdirAll(componentsDir, 0o755)
+		errDir := os.MkdirAll(resourcesDir, 0o755)
 		if errDir != nil {
 			return fmt.Errorf("error creating default components folder: %w", errDir)
```

Comment:
Update the error message to indicate resource directory instead of components folder, since components folder is marked for deprecation